### PR TITLE
Use error message const to avoid replication.

### PR DESCRIPTION
**Pull request by lzbj**
As we discussed before.. If we agree this is correct, I'll continue the remain ones.

### CODE REVIEW

**Comment by mrutkows** [comment]
As @rr (I believe) is indicating... 
https://github.com/apache/incubator-openwhisk/tree/master/tools/cli/go-whisk-cli/wski18n/resources

and specifically the translated map:
```
{
    "id": "Unable to create action '{{.name}}': {{.err}}",
    "translation": "Unable to create action '{{.name}}': {{.err}}"
},
```

**Comment by markusthoemmes** [comment]
Specifically the id in the translation file should be a well known constant, like

```
{
    "id": "ACTION_CREATE_UNABLE",
    "translation": "Unable to create action '{{.name}}': {{.err}}"
},
```

**Comment by lzbj** [comment]
Seems doesn't work like that, @mdeuser , could  you please help take a review?

File: tools/cli/go-whisk/whisk/wskerror.go [hunk 1]
```diff
@@ -34,6 +34,7 @@ const NO_DISPLAY_PREFIX         bool = false
 const APPLICATION_ERR           bool = true
 const NO_APPLICATION_ERR        bool = false
 const TIMED_OUT                 bool = true
+const UNABLE_CREATE_ACTION_ERR  string = "Unable to create action '{{.name}}': {{.err}}"
```
Comment (markusthoemmes) [inline tools/cli/go-whisk/whisk/wskerror.go]: This needs to be 

```go
const UNABLE_CREATE_ACTION_ERR  string = "UNABLE_CREATE_ACTION_ERR"
```

to work.

Comment (lzbj) [inline tools/cli/go-whisk/whisk/wskerror.go]: Ok, thank you for your help @markusthoemmes , seems I don't t get the real spirit of of the issue yet.

**Pull request closed without merging**